cadc-util/src/main/java/ca/nrc/cadc/date/DateUtil.java

```diff
@@ -328,10 +352,81 @@ public static Date toDate(final long dmfSeconds) {
         return new Date((dmfSeconds * 1000L) + getDMFEpoch().getTime());
     }
 
+    /**
+     * Convert from the given MJD value to a Java Date object.
+     *
+     * @param modifiedJulianDate    The MJD value.
+     * @return  Date instance.  Never null.
+     */
+    public static Date fromModifiedJulianDate(final double modifiedJulianDate) {
```

Comment:
This is a more complicated method that does the same thing as `Date toDate(double)`, which is already known to be correct (using `slaDjcl`). I appreciate that the existing method name is pretty vague (relying on the name of the arg to have any meaning, so it would be fine to deprecate it and add the more explicitly `fromModifiedJulianDate`. The deprecated one should call the new one and the new one should have the code from the existing one.

Can you also deprecate the `Date toDate(long)` and `Date getDMFEpoch()` methods?  I think "dmf" was an HST-specific thing that is not used in any of our s/w. Will remove in the future.